ui_src/src/components/tagList/style.scss

```diff
@@ -0,0 +1,77 @@
+.tags-list-wrapper {
+    display: flex;
+    align-items: center;
+    margin-left: 5px;
+
+    .plus-tags {
+        background-color: rgba(132, 129, 138, 0.1);
+        padding-right: 5px;
+        border-style: solid;
+        border-color: transparent;
+        border-width: thin;
+        padding-left: 5px;
+        border-radius: 4px;
+        height: 20px;
+        display: flex;
+        align-items: center;
+        cursor: pointer;
+        p {
+            font-family: 'InterSemiBold';
+            font-size: 12px;
+            line-height: 12px;
+            color: #2e2c34;
+            margin-top: 50%;
```

Comment:
When you use with percentage, should be responsive issues, so you need to be careful with that.